orchagent/notifications.cpp

```diff
@@ -86,3 +87,4 @@ sai_switch_notification_t switch_notifications
     on_switch_shutdown_request,
     NULL
 };
+*/
```

Comment:
in create switch, need to set attributes

SAI_SWITCH_ATTR_PORT_STATE_CHANGE_NOTIFY
SAI_SWITCH_ATTR_SHUTDOWN_REQUEST_NOTIFY